src/pages/EndpointsPage/model/EndpointItemViewModel.ts

```
@@ -38,48 +33,34 @@
-    return `/sandbox/${orgId}/${projectName}/${branchName}/${revisionTag}`
+  public get isGraphql(): boolean {
+    return this.item.type === EndpointType.Graphql
   }
 
-  public get linkUrl(): string {
-    return this.item.type === EndpointType.Graphql ? this.sandboxUrl : this.endpointUrl
+  public get sandboxUrl() {
+    if (this.isGraphql) {
+      return `${window.location.origin}/${SANDBOX_ROUTE}/${this.pathSegment}`
+    }
   }
```

Biome (2.1.2)

[error] This getter should return a value.



(lint/suspicious/useGetterReturn)